### ISSUE TITLE
Missing documentation of --isuptodate flag

### ISSUE THREAD

**Issue by erikbra** [erikbra/grate#582]
@erikbra is this flag missing now from documentation?

_Originally posted by @Tsingis in https://github.com/erikbra/grate/issues/504#issuecomment-2286086248_

**Comment by erikbra** [comment]
Yes it is, @Tsingis , you are absolutely correct. Would you like to make a shot at creating a PR?

**Comment by Tsingis** [comment]
#584 Here goes

**Comment by erikbra** [comment]
Fixed in #584
